### PR TITLE
omnibus 0.4.4 updates

### PR DESCRIPTION
**Pull request by sjmiles**

### CODE REVIEW

File: core/ts/utils/paths.ts [hunk 1]
```diff
@@ -7,6 +7,8 @@
  * https://developers.google.com/open-source/licenses/bsd
  */
 
+import { deepUndefinedToNull } from "./object";
```
Comment (mariakleiner) [inline core/ts/utils/paths.ts:10]: is this intentional? doesn't seem to be used

File: pkg/Library/App/Starter/arcs/StarterApp.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+ * https://developers.google.com/open-source/licenses/bsd
+ */
+
+import {App, logFactory, LocalStoragePersistor} from '../arcs.js';
+import {SimpleRecipe} from '../Library/StarterRecipe.js';
+
+const log = logFactory(true, 'SimpleApp', 'navy');
+
+// App class
+export const SimpleApp = class extends App {
```
Comment (mariakleiner) [inline pkg/Library/App/Starter/arcs/StarterApp.js:16]: `StarterApp` or `SimpleApp`? :)

**Review by mariakleiner — APPROVED** [review]

File: core/ts/utils/paths.ts (hunk 1 above)
Comment (sjmiles) [reply core/ts/utils/paths.ts:10]: Ty, no, that's something the IDE injected :(

File: pkg/Library/App/Starter/arcs/StarterApp.js (hunk 2 above)
Comment (sjmiles) [reply pkg/Library/App/Starter/arcs/StarterApp.js:16]: Eek, that whole folder is dross. Thanks again.

**Pull request merged**